### PR TITLE
CORE-8900 Add UI configs for Teams

### PR DESCRIPTION
**Pull request by ash3rz**

### CODE REVIEW

File: playbooks/de-populate-consul.yaml [hunk 1]
```diff
@@ -254,6 +254,9 @@
         - {key: "configs/{{environment_name}}/de/redis-port", value: "{{ redis_port }}"}
         - {key: "configs/{{environment_name}}/de/redis-db-number", value: "{{ redis_db_number }}"}
         - {key: "configs/{{environment_name}}/de/redis-password", value: "{{ redis_password }}"}
+        - {key: "configs/{{environment_name}}/de/grouper-id", value: "{{ apps.grouper_user }}"}
```
Comment (psarando) [inline playbooks/de-populate-consul.yaml]: This looks like a duplicate of `configs/{{environment_name}}/apps/grouper-user`.

The DE configs should probably just read from the existing `apps/grouper-user` key, unless we want to allow the DE and apps service to use different grouper user IDs.

**Review by psarando — CHANGES_REQUESTED** [review]
I think I caught a dup console config we might want to clean-up.
Otherwise everything else LGTM 👍

File: playbooks/de-populate-consul.yaml (hunk 1 above)
Comment (ash3rz) [reply playbooks/de-populate-consul.yaml]: Oops, you're right.  I saw the config there, I just didn't think to use it directly.

Comment (ash3rz) [reply playbooks/de-populate-consul.yaml]: I did a force push with the change.

Comment (ash3rz) [reply playbooks/de-populate-consul.yaml]: OK one more force push.

**Review by psarando — APPROVED** [review]

**Comment by ash3rz** [comment]
Thanks for the review!

**Pull request merged**